tests/RobotFramework/tests/cumulocity/shell/shell_operation.robot

```diff
@@ -13,6 +13,11 @@ Successful shell command with output
     Operation Should Be SUCCESSFUL    ${operation}
     Should Be Equal    ${operation.to_json()["c8y_Command"]["result"]}    helloworld\n
 
+Check Successful shell command with literal double quotes output
+    ${operation}=    Cumulocity.Execute Shell Command    echo \"helloworld\"
```

Comment:
RF is interpreting the single `\` as its escape char, and removing it from the called function (from the keyword). So we need to use a double backslash `\\`.

```suggestion
    ${operation}=    Cumulocity.Execute Shell Command    echo \\"helloworld\\"
```